contrib/inventory/zabbix.py

```diff
@@ -129,6 +129,8 @@ def __init__(self):
             try:
                 api = ZabbixAPI(server=self.zabbix_server, validate_certs=self.validate_certs)
                 api.login(user=self.zabbix_username, password=self.zabbix_password)
+            # zabbix_api tries to exit if it cannot parse what the zabbix server returned
+            # so we have to use BaseException here
             except BaseException as e:
```

Comment:
Catch `Exception` and `SystemExit` here instead of `BaseException`.

Same for other occurrences in the Zabbix module

Reply:
Roger wilco.